apps/e2ee-server/src/server.ts

```
@@ -0,0 +1,260 @@
+    IServerToClientEvents,
+    IInterServerEvents,
+    ISocketData
+  >;
+
+  private roomManager: RoomManager;
+
+  private config: IServerConfig;
+
+  constructor() {
+    this.config = {
+      port: parseInt(process.env.PORT || '3868', 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)